src/runtime/types.ts

```diff
@@ -548,6 +548,12 @@ export interface SecondarySignInOptions extends Record<string, unknown> {
    * @default false
    */
   external?: boolean
+  /**
+   * Whether `getSession` is called. If set to false, you have to manually call `getSession` to get the session data.
+   *
+   * @default true
+   */
+  withSession?: boolean
 }
 
 export interface SignUpOptions extends SecondarySignInOptions {
```

Comment:
This interface is odd IMHO. Because I never see this used in the code with any of the properties of the `SecondarySignInOptions` interface. It is just used next to a property which uses `SecondarySignInOptions`. But I could miss something which is there somewhere.

Reply:
You're actually right, it's a weird implementation in https://github.com/sidebase/nuxt-auth/pull/615 which accepts two identical interfaces:
https://github.com/sidebase/nuxt-auth/blob/218a8465fd01fbf70562ab4cc1083dea3931d1a1/src/runtime/composables/local/useAuth.ts#L161

It should be just `signUpOptions`, will probably move this to a different breaking change

Reply:
Issue created: https://github.com/sidebase/nuxt-auth/issues/948

Reply:
Excellent thanks 👍